conda_forge_tick/git_utils.xsh

```diff
@@ -152,7 +152,8 @@ def push_repo(feedstock_dir, body, repo, title, head, branch,
         deploy_repo = $USERNAME + '/' + $PROJECT + '-feedstock'
         doctr_run(['git', 'remote', 'add', 'regro_remote',
                    'https://{token}@github.com/{deploy_repo}.git'.format(
-                       token=token, deploy_repo=deploy_repo)])
+                       token=token, deploy_repo=deploy_repo)],
+                  token=token)
 
         git push --set-upstream regro_remote @(branch)
```

Comment:
you need to hide here too, in case the push fails, it will show your token

Reply:
Thank you!